test/remote/gateways/remote_element_test.rb

```diff
@@ -159,4 +159,13 @@ def test_transcript_scrubbing_with_echeck
     assert_scrubbed(@check.routing_number, transcript)
     assert_scrubbed(@gateway.options[:account_token], transcript)
   end
+
+  def test_successful_purchase_with_card_present_code
+    assert response = @gateway.purchase(@amount, @credit_card, @options.merge(card_present_code: 'Present'))
+    assert_success response
+    assert_equal 'Approved', response.message
+    assert_match %r{Street address and postal code do not match}, response.avs_result['message']
+    assert_match %r{CVV matches}, response.cvv_result['message']
```

Comment:
Since these two lines are checking things that are unrelated to this new field and are already covered by another test, I would remove them.

Reply:
The method now looks like this:
https://github.com/activemerchant/active_merchant/blob/548f0082dd3b100808794f0895231bcfb804d56f/test/remote/gateways/remote_element_test.rb#L53-L57